libs/asset-viewer/src/lib/components/asset-search-results/asset-search-results.component.html

```diff
@@ -1,31 +1,89 @@
-<ng-container *rxLet="assets$ | push">
-  <ng-container *rxLet="isResultsOpen$; let isResultsOpen">
-    <div class="header">
-      <div>
-        {{ "search.searchResults" | translate }}:
-        <asset-sg-animate-number [nextValue]="(pageStats$ | async)?.total ?? 0" />
-      </div>
-      <button asset-sg-icon-button style="width: unset" (click)="toggleResultsOpen(isResultsOpen)">
-        <span> {{ (isResultsOpen ? "search.hideTable" : "search.showTable") | translate }} </span>
-        <svg-icon key="close-nav"></svg-icon>
-      </button>
+<ng-container *rxLet="isResultsOpen$; let isResultsOpen">
+  <div class="header">
+    <div>
+      {{ "search.searchResults" | translate }}:
+      <asset-sg-animate-number [nextValue]="(pageStats$ | async)?.total ?? 0" />
     </div>
-    <ng-container *ngIf="isResultsOpen">
-      <div class="search-results">
-        <div *ngIf="(loadingState | async) === LoadingState.Loading; else results" class="loading-spinner">
-          <mat-progress-spinner mode="indeterminate" />
-        </div>
-        <ng-template #results>
-          <div
-            class="asset-result"
-            *rxFor="let asset of assets$ | async; trackBy: 'assetId'"
-            (mouseover)="onAssetMouseOver(asset.assetId)"
-            (mouseout)="onAssetMouseOut()"
-          >
-            <asset-sg-asset-search-result [asset]="asset"></asset-sg-asset-search-result>
-          </div>
-        </ng-template>
+    <button asset-sg-icon-button style="width: unset" (click)="toggleResultsOpen(isResultsOpen)">
+      <span> {{ (isResultsOpen ? "search.hideTable" : "search.showTable") | translate }} </span>
+      <svg-icon key="close-nav"></svg-icon>
+    </button>
+  </div>
+  <ng-container *ngIf="isResultsOpen">
+    <div class="search-results">
+      <div *ngIf="(loadingState | async) === LoadingState.Loading; else results" class="loading-spinner">
```

Comment:
Shouldn't we add/keep the error state, because in case an error occurs in the request, we will get the alert, but also the spinner will be there, which might be confusing

Reply:
I am now setting both laodingStates to loaded when the showAlert action is dispatched, not quite sure though if this is the right way to do it

Reply:
Setting the loadingState on `showAlert` is wrong, since the alert does not necessarily need to appear because of the search.

I would simply leave it as is and just remove the error state. If in the future, we want to be able to handle error states, we can just re-add it and fully implement it then. Right now, we don't have a specific solution on how to handle it.